pegasus/sites.v3/code.org/public/teacher-dashboard/index.haml

```diff
@@ -388,6 +398,86 @@ title: <%= I18n.t('dashboard_landing_title').inspect %>
 
   }]);
 
+  app.controller('MovingStudentsController', ['$scope', '$routeParams', '$q', '$window', '$http', 'sectionsService', function($scope, $routeParams, $q, $window, $http, sectionsService) {
+     var self = this;
+
+     // 'Other Section' selected
+     $scope.otherSection = 'Other Section';
+     $scope.stayEnrolledInCurrentSection = 'true';
+
+     // Query
+     $scope.currentSection = sectionsService.get({id: $routeParams.id});
+     $scope.sections = sectionsService.query();
+     $scope.students = sectionsService.allStudents({id: $routeParams.id});
+
+     // API post
+
+     $scope.moveStudents = function() {
+       // Need:
+       // new_section_code
+       // current_section_code
+       // stay_enrolled_in_current_section (kinda optional)
+       var params = {};
+       params['new_section_code'] = $scope.getNewSectionCode();
+       params['current_section_code'] = $scope.getCurrentSectionCode();
+       params['student_ids'] = $scope.getSelectedStudentIds();
+       params['stay_enrolled_in_current_section'] = $scope.getStayEnrolledInCurrentSection();
+       // TODO: Get this working
+       // $http.post($scope.transferUrl, params, {}).then(
+       sectionsService.moveStudents(params, {}).$promise.then(
+         function success(response) {
+           $('#move-students').modal('hide');
+         },
+         function error(response) {
+           debugger;
```

Comment:
@laurelfan I _think_ the route works, but I get an error when I try the API call: `422 (Unprocessable Entity)`. I checked online and it may be related to a lack of a CSRF token, but then I'd also be confused why other sectionsService calls aren't running into the same problem. Does that sound familiar to you? I'm going to keep digging into it.

Side question: I'm also running dashboard-server instead of pegasus since that's what I had started doing before the instructions were updated on the README page. Should I be using pegasus instead?